docs/dev-setup.md

```diff
@@ -27,10 +27,11 @@ install the operator-sdk tools.
     cd $GOPATH/src/github.com/metal3-io
     git clone https://github.com/metal3-io/baremetal-operator.git
     cd baremetal-operator
-    kubectl apply -f deploy/service_account.yaml
-    kubectl apply -f deploy/role.yaml
+    kubectl apply -f deploy/service_account.yaml -n metal3
+    kubectl apply -f deploy/role.yaml -n metal3
     kubectl apply -f deploy/role_binding.yaml
-    kubectl apply -f deploy/crds/metal3_v1alpha1_baremetalhost_crd.yaml
+    kubectl apply -f deploy/crds/metal3_v1alpha1_baremetalhost_crd.yaml -n metal3
```

Comment:
The CRD doesn’t need the namespace argument.

Reply:
Thanks for fixing this!

Can you squash the 2 commits together?  `git rebase -i origin/master`

Reply:
Sure. I have used Git in Web which is so difficult to use :(

Reply:
Done :)